npm/npm-debug-cli/dataplaneParser/iptable/iptable.go

```diff
@@ -0,0 +1,202 @@
+package iptable
+
+import (
+	"fmt"
+)
+
+type Iptables struct {
+	name   string
+	chains map[string]*IptablesChain
+}
+
+// Constructor
+func NewIptables(name string, chains map[string]*IptablesChain) *Iptables {
+	iptable := &Iptables{}
+	iptable.name = name
+	iptable.chains = chains
+	return iptable
+}
```

Comment:
these struct defs read as very java to me. to pick on one, is there a reason why your `IPTables` struct can't have its two fields exposed? then i'm able to use it as `ipt := &IPTables{name: "abc", chains: m}` instead of needing a constructor, getters, and setters